packages/connect-web/src/webusb/button.ts

```diff
@@ -19,16 +19,6 @@ const render = (className = '', url: string, origin = '') => {
             // btnIframe.style.opacity = '0'; // this makes click impossible on cross-origin
             btnIframe.setAttribute('allow', 'usb');
             btnIframe.setAttribute('scrolling', 'no');
-            btnIframe.onload = () => {
```

Comment:
since this is removed code in webusb/index should be:

~window.addEventListener('message', handleMessage);~
```
window.addEventListener('load', onLoad);
```